scipy/optimize/_lsq/least_squares.py

```diff
@@ -283,7 +283,7 @@ def least_squares(
         always uses the '2-point' scheme. If callable, it is used as
         ``jac(x, *args, **kwargs)`` and should return a good approximation
         (or the exact value) for the Jacobian as an array_like (np.atleast_2d
-        is applied), a sparse matrix (csr_matrix preferred for performance) or
+        is applied), a sparse matrix (csr_array preferred for performance) or
```

Comment:
Assume this will be changed in the doc pr
```suggestion
        is applied), a sparse array (csr_array preferred for performance) or
```